src/ApiClient.js

```diff
@@ -411,7 +411,7 @@ class ApiClient {
   hostSettings() {
     return [
       {
-        url: 'https://api.usepatch.com',
+        url: 'https://{defaultHost}',
```

Comment:
do we need to change this? 🤔

Reply:
You're right, we do not change this. I'll see about fixing this in the code generator repo...

Reply:
Have a PR up to fix this in the other repo!